chcemvediet/apps/inforequests/templates/admin/inforequests/action/delete_confirmation.html

```diff
@@ -1,13 +1,42 @@
 {% extends "admin/delete_confirmation.html" %}
-{% load amend prepend from poleno.amend %}
+{% load i18n %}
+{% load amend prepend before after delete from poleno.amend %}
 
 {% block content %}
   {% amend %}
     {{ block.super }}
+    {% after path="./ul[last()]" %}
+      {% if outbound %}
+        <p>Outbound messages will be deleted:</p>
+        <ul>{{ outbound|unordered_list }}</ul>
+      {% endif %}
+      {% if inbound %}
+        <p>Inbound messages will be marked undecided:</p>
+        <ul>{{ inbound|unordered_list }}</ul>
+      {% endif %}
+    {% endafter %}
+    {% if not object.is_last_action %}
+      {% before path=".//form" %}
+        <p>
+          <b>Warning:</b> The deleted action is not the last action in the branch. Deleting it may
+          cause logical errors in the inforequest history.
+        </p>
+      {% endbefore %}
+    {% endif %}
     {% if object.type == object.TYPES.EXPIRATION or object.type == object.TYPES.APPEAL_EXPIRATION %}
       {% prepend path=".//form" %}
         <p><label><input name="snooze" type="checkbox" /> Extend snooze of previous action by {{ ADMIN_EXTEND_SNOOZE_BY_DAYS }} days from today.</label></p>
       {% endprepend %}
     {% endif %}
+    {% if delete_dependency %}
+      {% before path=".//form" %}
+        <p class="errornote">Delete not allowed.</p>
+        <ul>{{ delete_dependency|unordered_list }}</ul>
```

Comment:
By som aj to `ul` dal do vnutra `errornote`. Nech je jasne, ze tie odrazky patria ku tej error hlaske:

```
<div class="errornote">
  <p>Delete not allowed.</p>
  <ul>{{ delete_dependency|unordered_list }}</ul>
</div>
```

Ak to teda vyrendruje spravne takto.

Reply:
![image](https://user-images.githubusercontent.com/29918752/114855691-85eef100-9de6-11eb-904c-a7e8e631f985.png)
@martinmacko47 Vyrendruje sa, len cerveny bold text v `<ul>` sa mi osobne nie velmi paci. 
:ok:

Reply:
IMHO good enough. Hlavne nech to je vyrazne.